src/smrt/shift_nek_driver.cpp

```diff
@@ -95,8 +95,8 @@ void ShiftNekDriver::solve()
   for (int iteration = 0; iteration < max_picard_iter_; ++iteration) {
     // Set heat source in Nek; first, get heat source indexed by cell ID
     // from Shift and map it to Nek elements
-    for (int cellid = 0; cellid < d_power_map.size(); ++cellid) {
-      const auto& elem_list = d_power_map[cellid];
+    for (int cellid = 0; cellid < d_shift_solver->d_power_map.size(); ++cellid) {
+      const auto& elem_list = d_shift_solver->d_power_map[cellid];
```

Comment:
This is fine for now.  I'm not a big fan of having data members being public and accessed like this.  Longer term, I think I would like to make the power map owned by `ShiftNekDriver`.  There would perhaps be a function in `ShiftDriver` that constructs and returns the map to be stored by `ShiftNekDriver`.